hack/update-metadata.sh

```
@@ -53,11 +56,12 @@ do
     if [ "${OCP_VERSION}" != "${PACKAGE_VERSION}" ]; then
         PACKAGE_VERSION="${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}"
     fi
+    MAX_OCP_VERSION=${MAX_OCP_VERSION:-"${MAJOR_VERSION}.$((MINOR_VERSION + 1))"}
 
     export NEW_CURRENT_CSV="${PACKAGE_NAME}.v${PACKAGE_VERSION}"
     export NEW_METADATA_NAME="${PACKAGE_NAME}.v${PACKAGE_VERSION}"
     export NEW_SKIP_RANGE=$(echo ${SKIP_RANGE} | sed "s/ <.*$/ <${PACKAGE_VERSION}/")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)